src/view/actions/sendEvent.jsx

```diff
@@ -174,138 +169,85 @@ const knownEventTypes = [
-                />
-              </div>
-            </div>
-            <div className="u-gapTop">
-              <InfoTipLayout
-                tip="The type of the experience event. Choose a predefined type or create
-                  your own. This will be added to the XDM object as the field `eventType`."
-              >
-                <FieldLabel labelFor="typeField" label="Type (optional)" />
-              </InfoTipLayout>
-              <div>
-                <WrappedField
-                  data-test-id="typeField"
-                  id="typeField"
-                  name="type"
-                  component={ComboBox}
-                  componentClassName="u-fieldLong"
-                  supportDataElement="replace"
-                  allowCreate
-                  options={knownEventTypes}
-                />
-              </div>
-            </div>
-            <div className="u-gapTop">
-              <InfoTipLayout
-                tip="Please specify a data element that will return a JavaScript
-                  object in XDM format. This object will be sent to the Adobe
-                  Experience Platform."
-              >
-                <FieldLabel labelFor="xdmField" label="XDM Data" />
-              </InfoTipLayout>
-              <div>
-                <WrappedField
-                  data-test-id="xdmField"
-                  id="xdmField"
-                  name="xdm"
-                  component={Textfield}
-                  componentClassName="u-fieldLong"
-                  supportDataElement="replace"
-                />
-              </div>
-            </div>
-            <div className="u-gapTop">
-              <InfoTipLayout
-                tip="The merge ID of the experience event. This will be added to
-                  the XDM object as the field `eventMergeId`."
-              >
-                <FieldLabel
-                  labelFor="mergeIdField"
-                  label="Merge ID (optional)"
-                />
-              </InfoTipLayout>
-              <div>
-                <WrappedField
-                  data-test-id="mergeIdField"
-                  id="mergeIdField"
-                  name="mergeId"
-                  component={Textfield}
-                  componentClassName="u-fieldLong"
-                  supportDataElement="replace"
-                />
-              </div>
-            </div>
-            <div className="u-gapTop">
-              <InfoTipLayout
-                tip="A platform experience event dataset ID that is different from the
-                dataset provided in the Edge configuration."
+          <Form>
+            <Picker
+              data-test-id="instanceNameField"
+              name="instanceName"
+              label="Instance"
+              items={getInstanceOptions(initInfo)}
+              width="size-5000"
+            >
+              {item => <Item key={item.value}>{item.label}</Item>}
+            </Picker>
```

Comment:
Yeah, good question. I left it off because there's technically no way for the user to deselect an instance name. On second thought, I think it's probably good to add it, especially since it might help the user to understand that the other fields are optional.